install_config/authentication/configuring_authentication_allow_all.adoc

```diff
@@ -0,0 +1,132 @@
+[[configuring-authentication-allow]]
+= Configuring the Allow All Identity Provider
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+
```

Comment:
For all of these topics that don't have multiple subtopics in them and therefore no need for a TOC, you can add the following here to make the spacing look a little better at the top of the page:

```
{nbsp} +
```

Or maybe just go ahead and add the `toc::[]` anyway as well, in case later some subtopics are added (the TOC won't render until there actually are subtopics):

```
toc::[]
{nbsp} +
```